newsfragments/aip-79.significant.rst

```diff
@@ -13,3 +13,13 @@ As part of this change the following breaking changes have occurred:
   - The property ``security_manager`` has been removed from the interface
 
   - The method ``filter_permitted_menu_items`` is now abstract and must be implemented
+
+* Types of change
```

Comment:
Should we add a type for this use case? I guess there must be several cases where an interface that user might implement have breaking changes. Something like "Code interface change". Users who have a custom auth manager or custom executor and implement the auth manager/executor interface will face breaking change. Having this category might help them. I dont expect many users doing it but I am pretty sure there are some.

Reply:
Sounds like a great idea! Let me update it!